pkg/placement/controller.go

```diff
@@ -75,17 +75,20 @@ var excludedResourceNames = map[string]bool{
 type Controller struct {
 	ctx              context.Context
 	logger           logr.Logger
-	ocmClient        client.Client
+	ocmClient        client.Client // TODO (maroon): this should be deleted when transport is ready
```

Comment:
transport is ready.. when do you think we should remove ocm client?
do you want to start testing using the updated transport?

Reply:
once this PR is in, I'll open another that removes the marked code and test e2e.